### PR TITLE
DOCS: New GUC for specifying address binding strategy

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Staging here: 

https://docs-staging.vmware.com/en/VMware-Tanzu-Greenplum/interconnect-address-guc/greenplum-database/GUID-ref_guide-config_params-guc-list.html#gp_interconnect_address_type

### CODE REVIEW

**Comment by soumyadeep2007** [comment]
1. Value range: `unicast, wildcard`
2. Set classifications should be: `local, session, reload`

Aside: I noticed that for gp_interconnect_type, the classifications are wrong. They should be:
`local, session, reload`

**Comment by VMWareJennyBeth** [comment]
@soumyadeep2007 Thanks for catching that error in gp_interconnect_type, Deep.

**Comment by soumyadeep2007** [comment]
I am not a 100% on the local vs master aspect, but the rest needs to change. I will get back about that.

**Review by soumyadeep2007 — APPROVED** [review]
And as Huansong pointed out, the default on 7X should be `unicast`.

Other than these aspects, the GUC description looks good!

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md [hunk 1]
```diff
@@ -950,7 +950,22 @@ The amount of shared memory, in kilobytes, allocated for query metrics. The defa
 |-----------|-------|-------------------|
 |integer `0 - 131072`|5120|master, system, restart|
 
-## <a id="gp_interconnect_debug_retry_interval"></a>gp\_interconnect\_debug\_retry\_interval 
+## <a id="gp_interconnect_address_type"></a>gp_interconnect_address_type
+
+Specifies the type of address binding strategy Greenplum database should use for communication between segment host sockets. There are two types: `unicast` and `wildcard`. The default is `wildcard`.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:955]: Global:  Greenplum database -> Greenplum Database

Comment (dyozie) [reply gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:955]: should use -> uses

File: gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md [hunk 2]
```diff
@@ -950,7 +950,22 @@ The amount of shared memory, in kilobytes, allocated for query metrics. The defa
...
+
+- When this parameter is set to `unicast`, Greenplum database  uses the `gp_segment_configuration.address` field to perform address binding. This reduces port usage on segment hosts and prevents interconnect traffic from being routed through unintended (and possibly slower) network interfaces. 
+
+- When this parameter is set to `wildcard`, Greenplum database uses a wildcard address for binding, enabling the use of any network interface compliant with routing rules.
+
+**NOTE**: In some cases, inter-segment communication using the unicast strategy may not be possible. One example is if the source segment's address field and the destination segment's address field are on different subnets and/or existing routing rules don't allow for such
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/config_params/guc-list.html.md:961]: don't -> do not

**Review by dyozie — CHANGES_REQUESTED** [review]

**Pull request merged**